examples/two-columns-layout/src/app.tsx

```diff
@@ -13,21 +14,28 @@ autocomplete({
   debug: true,
   autoFocus: true,
   openOnFocus: true,
-  plugins: [recentSearchesPlugin, querySuggestionsPlugin],
+  plugins: [recentSearchesPlugin, querySuggestionsPlugin, productsPlugin],
   render({ elements }, root) {
     const {
       recentSearchesPlugin: recentSearches,
       querySuggestionsPlugin: querySuggestions,
+      productsPlugin: products,
     } = elements;
 
+    const productsSection = products && (
+      <div className="aa-PanelSection--products">
+        <div className="aa-PanelSectionSource">{products}</div>
+      </div>
+    );
```

Comment:
Couldn't we inline this instead of storing it in a variable?

Reply:
I chose to store it in a variable because in the last PR, this section is displayed conditionally

Reply:
Couldn't we have the condition inlined?

Reply:
> Regarding the blurash placeholder, it will only show once Algolia has responded (not visible in your video). Until then we don't have the hash so we need to display something.

Is it possible that the state you mentioned is displayed at some point?
Items should not be visible until we get the results from Algolia, right?

Otherwise, we can simplify the states by just using a simple background color with a looping opacity animation when the image is not loaded and remove the Blurhash canvas, WDYT?